terraform/app/variables.tf

```diff
@@ -0,0 +1,92 @@
+variable environment {}
+
+
+# Gov.UK PaaS
+variable paas_api_url {
+}
+
+variable paas_password {
+  default = ""
+}
+
+variable paas_app_docker_image {
+  default = "ethanecf/ecf:1"
```

Comment:
This will change once we get a dockerhub account. We probably ought to make it private too